### DISCUSSION TITLE
Momento de Sabedoria #8

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#442] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/momento-de-sabedoria-8/

**Comment by brunoaugusto** [comment]
Porque eu tenho a impressão de que isso foi uma resposta ao [Momento de Sabedoria #7](</imagens/momento-de-sabedoria-7/>)?

**Comment by angellisy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [segunda-feira, 20 de junho de 2022](<https://www.vish.blog.br/imagens/momento-de-sabedoria-8/#comment_1160627076>):

> Porque eu tenho a impressão de que isso foi uma resposta ao [Momento de Sabedoria #7](</imagens/momento-de-sabedoria-7/>)?

![null](<https://i.imgur.com/KPepGG3.png>)

**Comment by one-eyed-willy** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [segunda-feira, 20 de junho de 2022](<https://www.vish.blog.br/imagens/momento-de-sabedoria-8/#comment_1160701401>):

> [![null](<https://camo.githubusercontent.com/62e2271d396fb36c7ae30f73590cc768a2a2233b66a7abd4003e2baa822ef9fb/68747470733a2f2f692e696d6775722e636f6d2f4b5065704747332e706e67>)](<https://camo.githubusercontent.com/62e2271d396fb36c7ae30f73590cc768a2a2233b66a7abd4003e2baa822ef9fb/68747470733a2f2f692e696d6775722e636f6d2f4b5065704747332e706e67>)

Eita :astonished:
